src/modules/nhl/tasks/LiveDataTask.ts

```
@@ -0,0 +1,130 @@
+    } else {
+      Stumper.error(`Failed to get live data for game ${gameId}`, "nhl:LiveDataTask:execute");
+    }
+  }
+
+  async sendPeriodNotification(threadId: string, period: number, type: PeriodType): Promise<void> {
+    const periodNotificationRoleId = ConfigManager.getInstance().getConfig("NHL")["livedata.periodNotificationRoleId"];
+    let message = `${roleMention(periodNotificationRoleId)}\n`;
+
+    switch (type) {
+      case PeriodType.REG:
+        let periodString = "";
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)